### PR TITLE
AEROGEAR-2119 - Security Self Defence Docs

### PR DESCRIPTION
**Pull request by rachael-oregan**
### Motivation

JIRA: https://issues.jboss.org/browse/AEROGEAR-2119

### Description

Documentation on how to use the security self defence checks.  This will need to be updated later once the metrics part is implemented in the SDK.

### Progress

- [x] Docs

### CODE REVIEW

**Comment by wtrocki** [comment]
👍

File: docs/security/README.adoc [hunk 1]
```diff
@@ -1,12 +1,88 @@
...
+= iOS Security - Self Defence Checks
 
-AeroGear Services security
+AeroGear iOS SDK offers a set of security features that will make sure that your application is meeting the modern OWASP recomendations. All security features are available as part of the separate `AGSSec` package. This package offers multiple security checks that can be used to analyse the security of the device. This can be useful for developers to detect vulnerablities and react accordingly. For example, if the app detects security issue *X* then do *Y*, where *X* could be no lock screen is set and *Y* is show a system prompt to setup the lock screen.
+
+This package offers:
+
+1. A suite of self-defence checks to detect an insecure or hostile environment.
+2. Integration with the AeroGear Services Metrics SDK to report self-defence checks data.
+3. Interfaces to access the results of self-defence checks and take appropriate action in application code.
```
Comment (wei-lee) [inline docs/security/README.adoc]: I think we should also mention that the package also allows developers to easily implement their own security checks and report metrics.

Comment (rachael-oregan) [reply docs/security/README.adoc]: I've added the custom security checks to this and the reporting the checks in metrics is already mentioned

File: docs/security/README.adoc [hunk 2]
```diff
@@ -1,12 +1,87 @@
...
 
 == Usage
 
-TODO: Add usage
+=== Adding Dependency
+
+Add dependency to your Podfile:
+
+```
+pod 'AGSSec'
```
Comment (StephenCoady) [inline docs/security/README.adoc]: based on comments on the PR I have open I have renamed this to AGSSecurity, so a few changes - sorry! Other than that this looks good to me.

**Pull request merged**